src/components/UserInfo.js

```diff
@@ -15,4 +16,8 @@ export default class UserInfo {
     this._nameElement.textContent = name;
     this._aboutElement.textContent = about;
   }
+
+  setUserAvatar(avatar) {
```

Comment:
It's great to have a setter for the avatar, since it's controlled by a different form